sdk/src/main/java/io/dapr/internal/opencensus/GrpcWrapper.java

```diff
@@ -0,0 +1,113 @@
+/*
+ * Copyright (c) Microsoft Corporation.
+ * Licensed under the MIT License.
+ */
+
+package io.dapr.internal.opencensus;
+
+import io.dapr.config.Property;
+import io.dapr.v1.DaprGrpc;
+import io.grpc.CallOptions;
+import io.grpc.Channel;
+import io.grpc.ClientCall;
+import io.grpc.ClientInterceptor;
+import io.grpc.ForwardingClientCall;
+import io.grpc.Metadata;
+import io.grpc.MethodDescriptor;
+import reactor.util.context.Context;
+
+import java.util.Map;
+import java.util.logging.Level;
+import java.util.logging.Logger;
+import java.util.stream.Collectors;
+
+/**
+ * Wraps a Dapr gRPC stub with telemetry interceptor.
+ */
+public final class GrpcWrapper {
```

Comment:
This is the only class in this package that is 100% written by us. I kept it here because it allows me to restrict access to most methods in this package.